Source/Locales/Menu/es.po

```diff
@@ -658,67 +657,67 @@ msgid "OK"
 msgstr "Aceptar"
 
 #: ../../Menu/Options.Designer.cs:308
-msgid "Brake pipe charging rate (PSI/s)             "
+msgid "Brake pipe charging rate (PSI/s)"
 msgstr "Velocidad de carga de la tubería de freno (PSI/s)"
 
 #: ../../Menu/Options.Designer.cs:319
-msgid "Graduated release air brakes                                "
+msgid "Graduated release air brakes"
 msgstr "Liberación gradual de los frenos de aire"
 
 #: ../../Menu/Options.Designer.cs:332 ../../Menu/TestingForm.Designer.cs:87
 msgid "Cancel"
 msgstr "Cancelar"
 
 #: ../../Menu/Options.Designer.cs:342
-msgid "Alerter in cab                                                        "
-msgstr "Alerta en la cabina"
+msgid "Alerter in cab"
+msgstr "\"Hombre Muerto\" en cabina"
 
 #: ../../Menu/Options.Designer.cs:355
-msgid "Control confirmations                                            "
+msgid "Control confirmations"
 msgstr "Confirmaciones de control"
 
 #: ../../Menu/Options.Designer.cs:367
-msgid "Map window                                                        "
+msgid "Map window"
 msgstr "Ventana del mapa"
 
 #: ../../Menu/Options.Designer.cs:433
 msgid "General"
 msgstr "General"
 
 #: ../../Menu/Options.Designer.cs:599
-msgid "Port number                                       "
+msgid "Port number"
 msgstr "Número de puerto"
 
 #: ../../Menu/Options.Designer.cs:630
-msgid "Enable webserver                                                "
+msgid "Enable webserver"
 msgstr "Habilitar el servidor web"
 
 #: ../../Menu/Options.Designer.cs:642
-msgid "Overspeed monitor                                                     "
+msgid "Overspeed monitor"
 msgstr "Monitor de exceso de velocidad"
 
 #: ../../Menu/Options.Designer.cs:654
-msgid "Disable TCS scripts                                              "
```

Comment:
This message was modified recently to "Enable TCS scripts", along with other changes. You may merge the upstream "master" branch into this "spanish-localization" branch and rebuild the translation files, to get the most updated strings.